src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -276,7 +276,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 assemblyIdentityComparer:=assemblyIdentityComparer,
                 strongNameProvider:=strongNameProvider,
                 metadataImportOptions:=metadataImportOptions,
-                referencesSupersedeLowerVersions:=referencesSupersedeLowerVersions)
+                referencesSupersedeLowerVersions:=referencesSupersedeLowerVersions,
+                generatedFilesOutputDirectory:=Nothing)
```

Comment:
@chsienki This compiler flag isn't supported for VB right?
Is there any work towards supporting it?

Reply:
This should be supported as we support VB source generators

Reply:
@jaredpar I can't see anything parallel to:

https://github.com/dotnet/roslyn/blob/a163778013dba0ad5efb32df5374584b0795b53a/src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs#L617

in https://github.com/dotnet/roslyn/blob/main/src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

The support for writing to disk was added in #47047. Was support for VB missed unintentionally?

Reply:
Yep, looks like we just missed it off when doing VB. Filed https://github.com/dotnet/roslyn/issues/51938 to track